### ISSUE TITLE
Window Git installer doesn't know when the @code User Installer has been installed.

### ISSUE THREAD

**Issue by auchenberg** [Microsoft/vscode#56848]
https://twitter.com/FreeRangeEggs/status/1031388009071955968

We should work with the Git team to enable this to work, especially as we are pushing people towards user installs.

**Comment by segevfiner** [comment]
This is the piece of code in the Git for Windows installer which needs to be updated: [git-for-windows/build-extra:1130-1139](https://github.com/git-for-windows/build-extra/blob/520aec965599f327e9ef9fcbd4aa7b45649749e8/installer/install.iss#L1130-L1139)

**Comment by dscho** [comment]
> This is the piece of code in the Git for Windows installer which needs to be updated: git-for-windows/build-extra:1130-1139

@segevfiner it is worse than that. The idea of having a system-wide installation of Git for Windows depend on a user-wide installation of VS Code is just... very questionable.

**Comment by segevfiner** [comment]
There is a per user config value for the editor. But I'm not sure if the
installer sets this in a per user config file or a system wide one.

בתאריך יום ג׳, 21 באוג׳ 2018, 13:36, מאת Johannes Schindelin ‏<
notifications@github.com>:

**Comment by dscho** [comment]
> There is a per user config value for the editor. But I'm not sure if the installer sets this in a per user config file or a system wide one.

Which installer do you mean?

**Comment by segevfiner** [comment]
The Git for Windows one. BTW, I checked and it does set the editor system
wide rather than per user.

This issue was closed anyhow since it's really an issue with the Git for
Windows installer rather than with VS Code.

**Comment by anthcool** [comment]
Just a heads up:
I try to update Git itself to version 2.19.0 from 2.18.0.windows.1 and am having this same problem.  If I try to update Git as described, using the installer downloaded from [https://git-scm.com/downloads](https://git-scm.com/downloads), and get to the default editor picker part, as soon as I go to choose VS Code as the default, or anything other than Vim, the Next button grays out/disables....  Ugh...

**Comment by jacekmusial** [comment]
I'm having similar issue on vanilla W10, with installed Visual Studio Code 1.27.2
Here is the issue for the git for windows itself https://github.com/git-for-windows/git/issues/1741 
tl;dr it's not fixed, yet.
